### PR TITLE
:book: Add collaboration announcement blog post

### PR DESCRIPTION
**Pull request by mudler**
on hold

### CODE REVIEW

**Review by mauromorales — APPROVED** [review]
pretty cool :sunglasses:

File: docs/content/en/blog/kairos-canonical.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+This is an exciting collaboration between the Kairos project, Spectro Cloud and Canonical to develop a solution that is highly performant, efficient, and scalable. The demos that have been presented at MWC showcase the advanced capabilities of the MicroK8s CAPI provider, and highlight the power of Kairos as a building block for distributed infrastructure substrates that can host even the most demanding modern OpenRAN o-DU, 5G UPF or AI/ML use-cases at scale.
+
+This is a true testament to the power of open-source technologies and community collaboration, and we can't wait to see what new possibilities this partnership will bring.
+
+Thank you for your continued support and enthusiasm for Kairos!
+
+**Details**: You can learn more about the Talco Radio Edge solution on the Ubuntu blog https://ubuntu.com/blog/meet-canonical-at-mwc-barcelona-2023 or at https://ubuntu.com/blog/canonical-at-mwc and watch it in action here: https://www.youtube.com/watch?v=wUCSK0O8Ro4
```
Comment (Itxaka) [inline docs/content/en/blog/kairos-canonical.md:23]: Not sure if this should be at the end of the page or encrusted higher in the post maybe?

File: docs/content/en/blog/kairos-canonical.md
Comment (Itxaka) [inline docs/content/en/blog/kairos-canonical.md:1]: In the ubuntu blog there is mention of Intel as well, should we mention it here too?

**Review by Itxaka — APPROVED** [review]
Good!

File: docs/content/en/blog/kairos-canonical.md (hunk 1 above)
Comment (mudler) [reply docs/content/en/blog/kairos-canonical.md:23]: :shrug: have no strong opinions, but didn't wanted to drive off the attention with action items in the middle of the text

**Pull request merged**